### PR TITLE
create admin

### PR DESCRIPTION
**Pull request by nghiabk**

### CODE REVIEW

File: app/views/admin/books/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<%= form_for [:admin,@book], html: {multipart: true}, remote: true do |f| %>
```
Comment (dieunb) [inline app/views/admin/books/_form.html.erb]: missing a space after ` [:admin,`

File: config/routes.rb [hunk 2]
```diff
@@ -8,4 +8,8 @@
   resources :categories
   resources :books
   resources :users
+
+  namespace :admin do
+    resources :categories, :books
```
Comment (dieunb) [inline config/routes.rb]: root ?

**Comment by HungSonSN** [comment]
ok

**Comment by tranhuyhoangbka** [comment]
ok

File: app/controllers/admin/categories_controller.rb [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    @category = Category.new category_params
+    if @category.save
+      flash[:success] = "Created success"
+      redirect_to categories_url
+    else
+      render 'new'
+    end
+  end
+
+  def new
```
Comment (chintk) [inline app/controllers/admin/categories_controller.rb]: new before create

File: app/controllers/admin/books_controller.rb [hunk 4]
```diff
@@ -0,0 +1,50 @@
+class Admin::BooksController < ApplicationController
+  before_action :load_categories, only: [:new, :edit]
+
+  def create
+    @book = Book.new book_params
+    if @book.save
+      respond_to do |format|
```
Comment (dieunb) [inline app/controllers/admin/books_controller.rb]:
```
+    if @book.save
+      redirect_to root_url, notice: "Book is created"
+    else
```

**Pull request closed without merging**